src/message.ts

```diff
@@ -42,7 +42,9 @@ export interface MsgRawObj {
   MMActualSender:   string, // getUserContact(message.MMActualSender,message.MMPeerUserName).isContact()
   MMPeerUserName:   string, // message.MsgType == CONF.MSGTYPE_TEXT && message.MMPeerUserName == 'newsapp'
   ToUserName:       string,
+  FromUserName:     string,
```

Comment:
`MsgRawObj` is the object from the WebWxApp.

Does  `FromUserName` and `Content` exist in the structure?

Just want to confirm that.